[反馈]: Bug 执行 npm run setup-auth 未自动安装依赖

### 部署平台

Linux Ubuntu

### 版本号

_No response_

### 客户端/工具

_No response_

### 请求接口

None

### 问题/请求描述

root@serve-1:~/AIStudioToAPI# npm run setup-auth

> aistudio-to-api@0.7.0 setup-auth
> node scripts/auth/setupAuth.js

node:internal/modules/cjs/loader:1386
  throw err;
  ^

Error: Cannot find module 'dotenv'
Require stack:
- /root/AIStudioToAPI/scripts/auth/setupAuth.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
    at Function._load (node:internal/modules/cjs/loader:1192:37)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)
    at Module.require (node:internal/modules/cjs/loader:1463:12)
    at require (node:internal/modules/helpers:147:16)
    at Object.<anonymous> (/root/AIStudioToAPI/scripts/auth/setupAuth.js:16:1)
    at Module._compile (node:internal/modules/cjs/loader:1706:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/root/AIStudioToAPI/scripts/auth/setupAuth.js' ]
}

Node.js v22.22.0

### 日志信息（可选）

```text

```

--- a/scripts/auth/setupAuth.js
+++ b/scripts/auth/setupAuth.js
@@ -12,9 +12,6 @@ const os = require("os");
 const path = require("path");
 const readline = require("readline");
 
-// Load environment variables from .env file
-require("dotenv").config({ path: path.resolve(__dirname, "..", "..", ".env") });
-
 const DEFAULT_CAMOUFOX_VERSION = "135.0.1-beta.24";
 const GITHUB_RELEASE_TAG_PREFIX = "v";
 
@@ -446,6 +443,10 @@ const ensureNodeModules = () => {
     execOrThrow(npmCommand(), ["install"], { cwd: PROJECT_ROOT, shell: true });
 };
 
+const loadEnvConfig = () => {
+    require("dotenv").config({ path: path.resolve(__dirname, "..", "..", ".env") });
+};
+
 const runSaveAuth = camoufoxExecutablePath => {
     console.log(getText("[4/4] 启动认证保存工具...", "[4/4] Starting auth save tool..."));
     console.log("");
@@ -497,6 +498,7 @@ const main = async () => {
     console.log("");
 
     ensureNodeModules();
+    loadEnvConfig();
 
     let camoufoxExecutablePath = process.env.CAMOUFOX_EXECUTABLE_PATH;
     if (!camoufoxExecutablePath) {

